src/main/java/org/elasticsearch/search/aggregations/bucket/InternalSingleBucketAggregation.java

```diff
@@ -64,6 +65,18 @@ public InternalAggregations getAggregations() {
     }
 
     /**
+     * Create a new copy of this {@link Aggregation} with the same settings as
+     * this {@link Aggregation} and contains the provided sub-aggregations.
+     * 
+     * @param subAggregations
+     *            the buckets to use in the new {@link Aggregation}
+     * @return the new {@link Aggregation}
+     */
+    public InternalSingleBucketAggregation create(InternalAggregations subAggregations) {
```

Comment:
I did think about that but the method in InternalMultiBucketAggregation is called create(InternalAggregations) so I wanted to make it the same as that one since they are equivalent